src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml.cs

```diff
@@ -143,6 +143,11 @@ public void FilterLabel_ControlSelected(object sender, EventArgs e)
             if (_selectedFilter != null)
             {
                 _selectedFilter.Selected = false;
+                if (_selectedFilter == _labelBrowse)
+                {
+                    // Clear the bitmapimagecache, which during browse could have grown to be large, only if we moved away from Browse.
+                    PackageItemListViewModel.ClearBitmapImageCache();
```

Comment:
The install list could still be large, and we probably don't want to reload again. I would assume we want something a bit more elaborate or just let it grow until we close the UI